.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      RAILS_ENV: test
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        ports:
+        - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set Ruby version to use
+        run: |
+          echo "::set-env name=RUBY_VERSION::$(cat .ruby-version)"
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "2.6.5"
+      - name: Install dependencies
+        run: |
+          sudo apt-get install libpq-dev
+          gem install bundler
+          bundle install --jobs 4 --retry 3
+      - name: Set up the test database
+        run: |
+          RAILS_ENV=test bundle exec rake db:setup
+        env:
+          DATABASE_URL: postgres://postgres:postgres@localhost:5432/team_dashboard_test
```

Comment:
This was the magic line that (when combined with the database.yml changes) fixed all our postgres connection issues.